source/support.tex

```diff
@@ -3166,6 +3168,214 @@
 An \impldef{return value of \tcode{bad_typeid::what}} \ntbs{}.
 \end{itemdescr}
 
+\rSec1[support.srcloc]{Class \tcode{source_location}}
+
+The header \tcode{<source_location>} defines the class \tcode{source_location} that provides a means to obtain source location information.
+
+\rSec2[source_location.syn]{Header \tcode{<source_location>} synopsis}
+
+\begin{codeblock}
+namespace std {
+  struct source_location {
+    // source location construction
+    static consteval source_location current() noexcept;
+    constexpr source_location() noexcept;
+
+    // source location field access
+    constexpr uint_least32_t line() const noexcept;
+    constexpr uint_least32_t column() const noexcept;
+    constexpr const char* file_name() const noexcept;
+    constexpr const char* function_name() const noexcept;
+
+  private:
+    uint_least32_t line_;               // \expos
+    uint_least32_t column_;             // \expos
+    const char* file_name_;             // \expos
+    const char* function_name_;         // \expos
+  };
+}
+\end{codeblock}
+
+\pnum
+The type \tcode{source_location} meets the
+\oldconcept{DefaultConstructible},
+\oldconcept{CopyConstructible},
+\oldconcept{Copy\-Assignable}, and
+\oldconcept{Destructible}
+requirements\iref{utility.arg.requirements}.
+Lvalues of type \tcode{source_location}
+are swappable\iref{swappable.requirements}.
+
+\pnum
+All of the following conditions are true:
+\begin{itemize}
+\item \tcode{is_nothrow_move_constructible_v<source_location>}
+\item \tcode{is_nothrow_move_assignable_v<source_location>}
+\item \tcode{is_nothrow_swappable_v<source_location>}
+\end{itemize}
+\begin{note}
+The intent of \tcode{source_location} is
+to have a small size and efficient copying.
+\end{note}
```

Comment:
The paper wants this in a separate paragraph, and I agree - it has to do with the class, not the conditions mentioned in the paragraph it's currently in.

Reply:
I disagree. The note wants to say that those "nothrow-copy" rules are conducive to the goal in the note. Again, I want to avoid single-sentence paragraphs.